testsuite/podman_runner/02_setup_network.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 set -ex
-podman network create uyuni-network-1
+sudo -i podman network create --ipv6 --subnet 2001:db8::/64 uyuni-network-1
```

Comment:
That might need to be parameterizable maybe.

Reply:
Maybe, the thing is that I do not understand very much which subnets can you use...

Reply:
I just added the optional parameter "s" for setting a different subnet. If the parameter is not set, it uses the default 2001:db8::/64